Coordinators test

Lyonsclay opened the pull request.

Lyonsclay committed:
added coordinators_controller.rb

Lyonsclay committed:
not sure where changes came in

Lyonsclay committed:
fixed coordinator_visits_dashboard test

kexline4710 commented:
Awesome work on the integration test Clay! You are the :bomb: 

Can you just fix the merge conflicts in the seeds file really quick? Then it should be good to go!

Lyonsclay committed:
fixing seed file conflict
Merge branch 'master' of https://github.com/kexline4710/PatientEngagement into coordinators_test

Lyonsclay committed:
merge conflict with seeds
